migration/sql-files/062-workitem-related-changes.sql

```diff
@@ -0,0 +1,77 @@
+-- add a column to record the timestamp of the latest addition/change/removal of a comment on a workitem
+ALTER TABLE work_items ADD COLUMN commented_at timestamp with time zone;
```

Comment:
Why do you add this column? Isn't it redundant and the same as the `updated_at` column of the last updated comment for this work item?